boxes/tasks.py

```diff
@@ -2,20 +2,34 @@
 from django.core.mail import EmailMultiAlternatives
 from django.conf import settings
 from django.utils import timezone
+
 from .models import Message
+from .email import GPGSignedEncryptedMessage
 
 from celery import shared_task
 
+import gnupg
 
 @shared_task
 def process_email(message_id, form_data):
     message = Message.objects.get(id=message_id)
     box = message.box
+    msg = form_data["message"]
     subject = "New submission to your box: {}".format(box)
-    # TODO SignMessage Here
-    email = EmailMultiAlternatives(subject, form_data["message"],
-                                   settings.DEFAULT_FROM_EMAIL,
-                                   [box.owner.email])
+
+    if box.owner.server_signed:
+      email = GPGSignedEncryptedMessage(subject, msg,
+                                      settings.DEFAULT_FROM_EMAIL,
+                                      [box.owner.email])
+    else:
+      email = EmailMultiAlternatives(subject, msg,
```

Comment:
Both of the lines where `email` is assigned, do not respect the 4 indentation spaces, used on the remaining of the project.